libsolidity/formal/SSAVariable.h

```diff
@@ -80,10 +80,10 @@ class SSAVariable
 	}
 
 	std::shared_ptr<SymbolicVariable> m_symbolicVar = nullptr;
-	int m_currentSequenceCounter;
+	int m_currentSequenceCounter{};
```

Comment:
This probably makes sense to be initialised to 0, either here or in the constructor initialiser list.

Reply:
yep!

Reply:
the thing is it will be initialized within `resetIndex()`..